### PR TITLE
Opt include

### PR DESCRIPTION
**Pull request by cbielow**
augments #4381

fix behaviour of TOPPBase InputFile tags ('is_executable' and 'skipexists').
They are checked now more frequently, i.e. if the value is required or non-empty.
The checks happens when `getStringOption_()` is called.

### CODE REVIEW

File: src/utils/MetaProSIP.cpp [hunk 1]
```diff
@@ -3331,33 +3331,21 @@ class TOPPMetaProSIP :
       {
         if (isotopic_intensities[i] < 1e-4) continue;
         Size consecutive_isotopes = 0;
-        Int j = i;
+        Size j = i;
 
-        while (j >= 0)
+        while (j != -1)
```
Comment (jpfeuffer) [inline src/utils/MetaProSIP.cpp]: this fails I think.

Comment (cbielow) [reply src/utils/MetaProSIP.cpp]: yeah... compiler warnings... fixed.

**Comment by cbielow** [comment]
rebuild jenkins

**Comment by cbielow** [comment]
checks pass. merge?

**Pull request merged**